arch/arm/boot/dts/armada-385-ts7800-v2.dts

```diff
@@ -398,3 +549,22 @@
 &usb3_1 {
 	status = "okay";
 };
+
+&gpio0 {
+	status = "okay";
+
+	spi0_cs3n {
+		gpio-hog;
+		gpios = <28 0x0>;
+		output-high;
+		line-name = "spi0-cs3n";
+	};
+
+	en_usb_host_5v {
+		gpio-hog;
+		gpios = <45 0x0>;
+		output-high;
+		line-name = "en-usb-host-5v";
+	};
```

Comment:
I'm not opposed to this necessarily, however using gpio-hog; prevents userspace from accessing these GPIO. USB 5V would be always asserted. And I'm not sure what happens with the SPI CS# if the kernel can still make use of it or not.

Reply:
I don't have any reason to keep them gpio-hog except that's how they were when I pulled it into the branch.

Reply:
In that case, unless there is a known specific reason, they should be configured without the hog. Configured in the iomux with a pull resistor if necessary (or possible on that CPU).